server/routes/reporting/standard-report/definitions.js

```diff
@@ -1343,6 +1343,24 @@
           totals: true,
         }
       },
+      'unpaid-attendance-detailed': {
+        title: 'Unpaid attendance report (detailed)',
+        apiKey: 'UnpaidAttendanceReportDetailedReport',
+        search: 'dateRange',
+        printLandscape: true,
```

Comment:
On Figma the document is printed Portrait but you have Landscape?

Reply:
Changed